### ISSUE TITLE
Support 7.07?

### ISSUE THREAD

**Issue by GeraldLee** [andreiapostoae/dota2-predictor#14]
will you guys support version 7.07? IceFrog added two new heroes.
btw, how you guys get the range of match id for a specific version of dota2.

**Comment by andreiapostoae** [comment]
I do plan to keep this tool updated, however I have been busy working for my BSc thesis.
Getting the match ID for a patch is not done programatically as the patch releases are rare and instead I check the patch release date and simulate a binary search on dotabuff match IDs until I hit the date of the release. If the patch is older and dotabuff does not show the actual date of the game, I use opendota the same way. 

Basically:
1. find the release date of a patch (info can be found online)
2. find the last day of a patch (when the next patch is released or today if it is the latest patch)
3. use dotabuff/opendota to query games and compare UNIX timestamp until those two dates are hit and then save the IDs in the [config file](https://github.com/andreiapostoae/dota2-predictor/blob/master/metadata.json) (check bottom of the file)
